### PR TITLE
Adding Label 10 parsing

### PR DESCRIPTION
**Pull request by makrsmark**
LDR, POS, and / messages

### CODE REVIEW

**Comment by makrsmark** [comment]
@rpatel3001 - i can't add you as a reviewer, but i'd like you to take a look

**Comment by rpatel3001** [comment]
I can't review too much of the format itself because i guess airframes is down and i don't have some of these formats in my local logs. I think a matching PR to the docs repo would be helpful, and maybe anything that pushes onto the formatted results array should go into result_formatter.

**Comment by kevinelliott** [comment]
> I can't review too much of the format itself because i guess airframes is down and i don't have some of these formats in my local logs. I think a matching PR to the docs repo would be helpful, and maybe anything that pushes onto the formatted results array should go into result_formatter.

Yeah it's down for maintenance right now. Last of the big bug fix maintenances we've been doing.

File: lib/MessageDecoder.ts [hunk 1]
```diff
@@ -15,6 +15,9 @@ export class MessageDecoder {
 
     this.registerPlugin(new Plugins.Label_ColonComma(this));
     this.registerPlugin(new Plugins.Label_5Z(this));
+    this.registerPlugin(new Plugins.Label_10_LDR(this));
```
Comment (kevinelliott) [inline lib/MessageDecoder.ts:18]: Do we have some idea of what LDR is? I think we might want to start putting comments at the very top of each decoder to give some sense of what it is for. Brief summary.

File: lib/plugins/Label_10_Slash.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      decodeResult.remaining.text = message.text;
+      decodeResult.decoded = false;
+      decodeResult.decoder.decodeLevel = 'none';
+      return decodeResult;
+    }
+
+    const lat = parts[1];
+    const lon = parts[2];
+    const position = {
+      latitude: (lat[0] === 'N' ? 1 : -1) * Number(lat.substring(1)),
```
Comment (kevinelliott) [inline lib/plugins/Label_10_Slash.ts:39]: I think we could start extracting this kind of stuff out to the utility functions perhaps?

**Review by kevinelliott — APPROVED** [review]
Looks good to me. Just some comments, but nothing blocking merge.

File: lib/MessageDecoder.ts (hunk 1 above)
Comment (makrsmark) [reply lib/MessageDecoder.ts:18]: no idea - If i did, i'd name it something better than "position report"

File: lib/plugins/Label_10_Slash.ts (hunk 2 above)
Comment (makrsmark) [reply lib/plugins/Label_10_Slash.ts:38]: yes but... all of these messages are slightly different i run into "naming things" issues... totally open to suggestions. I would put it in `CoordinateUtils`

**Comment by makrsmark** [comment]
@rpatel3001 good catch! I was missing adding the raw position, which would make this decoding worthless to acars2pos

**Pull request merged**